proto/identity/api/v1/identity.proto

```diff
@@ -51,31 +50,35 @@ message PublishIdentityUpdateRequest {
   xmtp.identity.associations.IdentityUpdate identity_update = 1;
 }
 
+message PublishIdentityUpdateResponse {
+  string inbox_id = 1;
+}
+
 // Get all updates for an identity since the specified time
 message GetIdentityUpdatesRequest {
   // Points to the last entry the client has received. The sequence_id should be
   // set to 0 if the client has not received anything.
-  message XidCursor {
-    string xid = 1;
+  message InboxCursor {
```

Comment:
Should we actually just call this `Request`, like on the other endpoints? I fear `InboxCursor` might be interpreted as a cursor to the messages in your inbox.